HW5_OlhaMaraduda/HW5_Task3_Olha_Maraduda.py

```diff
@@ -0,0 +1,13 @@
+# TASK_3
+# Write the script that will calculate the factorial of the entered number without using recursion.
+# Example: 0!=1, 1!=1, 2!=2, 3!=1*2*3=6.
+
```

Comment:
good job
look at this version of code 
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)